grails-app/views/replenishment/print.gsp

```diff
@@ -27,7 +27,7 @@
     <hr/>
 </div>
 
-<g:set var="enablePageBreak" value="${params.pageBreak != 'Disable page break'}"/>
+<g:set var="enablePageBreak" value="${params.pageBreak == 'Enable page break'}"/>
```

Comment:
I changed it like that, because Katarzyna said the page break should be disabled by default. For now we can select:
- "" (this one is selected be default)
- "Enable page break"
- "Disable page break"
So previously, when it was `params.pageBreak != 'Disable page break'` the `enablePageBreak` was set to `true` by default.

Reply:
:+1: